### DISCUSSION TITLE
Culling idle kernels - documentation consistency

### DISCUSSION THREAD

**Discussion by bsdz** [jupyter/enterprise_gateway#1342] [Q&A]
Hi. 

I'm trying to enable the culling of idle kernels as described under:

https://jupyter-enterprise-gateway.readthedocs.io/en/main/operators/config-culling.html

In the example given, I notice an inconsistency in the description which uses a `RemoteKernelManager` prefix for `cull_idle_timeout`, `cull_interval` and `cull_connected`; whereas the sample bash script shows `MappingKernelManager.cull_interval` (i.e. `MappingKernelManager`).

To add to the confusion, I see in some scripts within the git repo references to `RemoteMappingKernelManager`, e.g.

https://github.com/jupyter-server/enterprise_gateway/blob/46a214c342a75cd599b175a0ee2ffa646b8de48b/etc/docker/enterprise-gateway/start-enterprise-gateway.sh#L46

Are all these interchangable, `RemoteMappingKernelManager`, `RemoteKernelManager` and `MappingKernelManager`? I ask as currently I am using `RemoteKernelManager` and it does not appear to culling my remote disconnected kernels with in my configured time of 12h.

I'm happy to update the documentation if it's got out of sync with the code, ie should say `RemoteMappingKernelManager`.

**Comment by kevin-bates** [comment]
Hi @bsdz - thanks for bringing this to our attention.  Yes, the reference to `RemoteKernelManager` is incorrect and should be updated to `RemoteMappingKernelManager`.   (I believe these settings may have been moved to `MappingKernelManager` - having been monkey-patched onto `KernelManager` previously.)

Regarding interchangeability, traits can be enabled from any subclass, up the hierarchy, to the point at which they are defined.  This is why you find use of `RemoteMappingKernelManager` and `MappingKernelManager` both work.  However, because `RemoteMappingKernelManager` is locally defined in EG, I think its probably best that we be consistent (with respect to EG) and always use `RemoteMappingKernelManger` when referencing traits  defined on `MappingKernelManager`.

Aside from the incorrect reference to `RemoteKernelManager` in docs/source/operators/config-culling.md, there are references to `MappingKernelManager` that should also be updated to `RemoteMappingKernelManager` in these file: 
docs/source/operators/config-culling.md
docs/source/operators/launching-eg.md
etc/docker/enterprise-gateway-demo/start-enterprise-gateway.sh.template

It would be really great if you could provide a pull request to update these locations.
